lib/db/mysql/index.js

```diff
@@ -108,7 +108,7 @@ MysqlStore.connect = function mysqlConnect(options) {
   options.mysql = mysql;
   var patcher = new MysqlPatcher(options);
 
-  return P.promisify(patcher.connect, patcher)().then(function() {
+  return P.promisify(patcher.connect, {context: patcher})().then(function() {
```

Comment:
per http://bluebirdjs.com/docs/new-in-bluebird-3.html#promisification-api-changes